joomla/joomla-cms
# [4.0] Cleanup the View files com_admin

### Summary of Changes
This is a first step in cleaning up the view files and adhering to coding standards as well as making the views more usable by the IDE by removing the magic getter.


### Testing Instructions
1. Go to System -> System Information
2. Check that the page still works
3. Click on the `Download as text`
4. Check that the file is generated
5. Click on the `Download as JSON`
6. Check that the file is generated
7. Click on Help in the left menu
8. Click on Joomla! Help
9. Check that the page still shows


### Documentation Changes Required
None
